test/commands/passphrase/decrypt.test.js

```diff
@@ -0,0 +1,157 @@
+				getInputsFromSources,
+				'default',
+				sandbox.stub().resolves(defaultInputs),
+			);
+
+	describe('passphrase:decrypt', () => {
+		setupTest()
+			.stdout()
+			.command(['passphrase:decrypt'])
+			.catch(error =>
+				expect(error.message).to.contain(
+					'No encrypted passphrase was provided.',
+				),
+			)
+			.it('should throw an error');
+	});
+
+	describe('passphrase:decrypt encryptedPassphrase', () => {
+		setupTest()
+			.stdout()
+			.command(['passphrase:decrypt', defaultEncryptedPassphrase])
+			.it('should decrypt passphrase with arg', () => {
+				expect(getInputsFromSources.default).to.be.calledWithExactly({
+					password: {
+						source: undefined,
+					},
+					data: null,
+				});
+				expect(cryptography.parseEncryptedPassphrase).to.be.calledWithExactly(
+					defaultEncryptedPassphrase,
+				);
+				expect(
+					cryptography.decryptPassphraseWithPassword,
+				).to.be.calledWithExactly(
+					encryptedPassphraseObject,
+					defaultInputs.password,
+				);
+				return expect(printMethodStub).to.be.calledWithExactly({ passphrase });
+			});
+	});
+
+	describe('passphrase:decrypt --passphrase=file:./path/to/encrypted_passphrase.txt', () => {
+		const passphraseSource = 'file:./path/to/encrypted_passphrase.txt';
+		setupTest()
+			.stdout()
+			.command(['passphrase:decrypt', `--passphrase=${passphraseSource}`])
+			.it('should decrypt passphrase with passphrase flag', () => {
+				expect(getInputsFromSources.default).to.be.calledWithExactly({
+					password: {
+						source: undefined,
+					},
+					data: {
+						source: passphraseSource,
+					},
+				});
+				expect(cryptography.parseEncryptedPassphrase).to.be.calledWithExactly(
+					defaultEncryptedPassphrase,
+				);
+				expect(
+					cryptography.decryptPassphraseWithPassword,
+				).to.be.calledWithExactly(
+					encryptedPassphraseObject,
+					defaultInputs.password,
+				);
+				return expect(printMethodStub).to.be.calledWithExactly({ passphrase });
+			});
+	});
+
+	describe('passphrase:decrypt --passphrase=filePath --password=pass:456', () => {
+		const passphraseSource = 'file:./path/to/encrypted_passphrase.txt';
+		setupTest()
+			.stdout()
+			.command([
+				'passphrase:decrypt',
+				`--passphrase=${passphraseSource}`,
+				'--password=pass:456',
+			])
+			.it(
+				'should decrypt passphrase with passphrase flag and password flag',
+				() => {
+					expect(getInputsFromSources.default).to.be.calledWithExactly({
+						password: {
+							source: 'pass:456',
+						},
+						data: {
+							source: passphraseSource,
+						},
+					});
+					expect(cryptography.parseEncryptedPassphrase).to.be.calledWithExactly(
+						defaultEncryptedPassphrase,
+					);
+					expect(
+						cryptography.decryptPassphraseWithPassword,
+					).to.be.calledWithExactly(
+						encryptedPassphraseObject,
+						defaultInputs.password,
+					);
+					return expect(printMethodStub).to.be.calledWithExactly({
+						passphrase,
+					});
+				},
+			);
+	});
```

Comment:
What about when both come from stdin?

Reply:
stdin should be covered by two flags case because it's stubbed.

Reply:
OK good, and the order is covered by utils tests.